packages/compass-aggregations/src/components/pipeline-toolbar/pipeline-settings/hidden-on-narrow-container.ts

```diff
@@ -0,0 +1,8 @@
+import { css } from '@mongodb-js/compass-components';
+import { workspaceContainerQueryName } from '@mongodb-js/compass-components';
```

Comment:
I was thinking about what we discussed yesterday and to make it more contextual, how do you feel about assigning this property to the component constructor? Maybe a good way to make it super obvious what the container query refers to?

```js
import { WorkspaceContainer } from '@mongodb-js/compass-components';

css`
  @container ${WorkspaceContainer.containerQueryName} (...) { ... }
`
```

Super nit though, feel free to ignore, just thinking how to make it  not just a random-ish const export from the components package

Reply:
Makes sense!

Actually, I was almost about to remove this from WorkspaceContainer :). While doing this I stumbled on a couple of controls for which, if we want to do something nice we may still need something for react, cause hiding labels with css on LG controls is not the easiest thing ever :(